app/controllers/api/database.php

```diff
@@ -1145,6 +1145,9 @@
         catch (StructureException $exception) {
             throw new Exception($exception->getMessage(), 400);
         }
+        catch (DuplicateException $exception) {
+            throw new Exception($exception->getMessage(), 400);
```

Comment:
Should we use 400 or 409 for this error?

Reply:
409 makes sense 👍🏻 will update